ci/run_ctests.sh

```diff
@@ -6,5 +6,12 @@ set -euo pipefail
 # Support customizing the ctests' install location
 cd "${INSTALL_PREFIX:-${CONDA_PREFIX:-/usr}}/bin/tests/libkvikio/"
 
-# Run BASIC_IO_TEST
-./BASIC_IO_TEST
+# Run basic tests
+rapids-logger "Run BASIC_IO_EXAMPLE"
+./BASIC_IO_EXAMPLE
+rapids-logger "Run BASIC_NO_CUDA_EXAMPLE"
+./BASIC_NO_CUDA_EXAMPLE
+
+# Run gtests
+rapids-logger "Run gtests"
```

Comment:
I have a weak preference to avoid `rapids-logger` commands in the `run_*.sh` scripts because I think those are intended for reuse outside of the CI workflows (in DLFW, perhaps?). In other use cases, the `gha-tools` utilities may not be present. Either use `echo` or replace the `rapids-logger` statements with code comments.

Reply:
@trxcllnt may have comments about appropriate use cases here.

Reply:
I have removed the rapids-logger use